Core/tests/Unit/Batch/SysvConfigStorageTest.php

```diff
@@ -68,11 +68,67 @@ public function testSaveBadConfig()
         } catch (\RuntimeException $e) {
             // verify we didn't corrupt memory
             $this->storage->load();
+            // Just to avoid the test warning
+            $this->assertTrue(true);
             return;
         }
 
         $this->assertTrue(false, 'should have thrown an exception');
     }
+
+    public function testDefaultValues()
+    {
+        if (getenv('GOOGLE_CLOUD_BATCH_SHM_SIZE')
+            || getenv('GOOGLE_CLOUD_BATCH_PERM')
+            || getenv('GOOGLE_CLOUD_BATCH_PROJECT')) {
+            $this->markTestAsSkipped(
+                'Remove GOOGLE_CLOUD_BATCH_* env vars to run this test'
+            );
+        }
+        $r = new \ReflectionObject($this->storage);
+        $p = $r->getProperty('shmSize');
+        $p->setAccessible(true);
+        $this->assertEquals(200000, $p->getValue($this->storage));
+        $p = $r->getProperty('perm');
+        $p->setAccessible(true);
+        $this->assertEquals(0600, $p->getValue($this->storage));
+        $p = $r->getProperty('project');
+        $p->setAccessible(true);
+        $this->assertEquals('A', $p->getValue($this->storage));
+    }
+
+    public function testEnvVarCustomization()
+    {
+        $originalShmSize = getenv('GOOGLE_CLOUD_BATCH_SHM_SIZE');
```

Comment:
Can probably do this in the setup/teardown.

Reply:
Done